lib/teracy-dev/extension/manager.rb

```diff
@@ -52,13 +53,22 @@ def sync(extension)
           "path" => path
         })
         sync_existing = extension['path']['lookup'] == DEFAULT_EXTENSION_LOOKUP_PATH
-        Location::Manager.sync(extension['location'], sync_existing)
+
+        if Util.cmdRequireSync?
+          Location::Manager.sync(extension['location'], sync_existing)
+        end
+
       end
 
       def validate(extension)
         return unless Util.true?(extension['enabled'])
 
-        manifest = Manager.manifest(extension)
+        begin
+          manifest = Manager.manifest(extension)
+        rescue Errno::ENOENT
+          @logger.error("manifest for extension #{extension['path']['extension']} not found, seem you not sync all required extension. Please run vagrant <status|up|reload> to sync it first.")
```

Comment:
@logger.error("manifest for extension #{extension['path']['extension']} not found, seem that you haven't synced all the required extensions. Please run vagrant <status|up|reload> to sync them first")